notifications_utils/safe_string.py

```diff
@@ -0,0 +1,20 @@
+import re
+import unicodedata
+
+
+def email_safe(string, whitespace='.'):
```

Comment:
Optional: I'm not that keen on the wording of these two functions. I can see that email safe is taking a word and getting rid of whitespace, unicode and replacing whitespace with `.` but I can't quite see how that relates to being 'email_safe'. Aware that there would be work required to change the name and may be out of scope so I'll let you decide. Similiar with `id_safe` which doesn't mean much either

Another optional refactor would be to have a base function such that:

```
def make_my_string_in_a_certain_format(string, whitespace):
    # all the code

def email_safe(string):
    return make_my_string_in_a_certain_format(string, whitespace='.')

def id_safe(string):
    return make_my_string_in_a_certain_format(string, whitespace='-')
```